### ISSUE TITLE
Not admitted body in http request

### ISSUE THREAD

**Issue by DavideDA** [kvs8/vedro-replay#7]
Hello!
I am trying to submit a request like this to vedro-replay

```
### My post request
POST https://{{host}}/api/books
Content-Type: application/json

[1]
```

but vedro refuses to accept it:

```
File "/workspaces/ceb-regression-testing/scenarios/api.py", line 9, in <module>
    class Scenario(vedro.Scenario):
  File "/workspaces/ceb-regression-testing/scenarios/api.py", line 12, in Scenario
    @replay("requests/api.http")
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/vedro_replay/replay.py", line 13, in wrapped
    for request in reversed(parse_requests(requests_file)):
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/vedro_replay/parse_requests.py", line 116, in parse_requests
    raise IncorrectContentsRequestFile(f"Failed to process file contents {requests_file}") from e
vedro_replay.parse_requests.IncorrectContentsRequestFile: Failed to process file contents requests/api.http
```

I am pretty sure that the json body I pass is correct: Java Play Framework admits it and Spring Boot too.
I think vedro-replay parser is working wrong.
Thank you.

**Comment by DavideDA** [comment]
Hello again.
I have tried to replace in file parse_requests.py line 73 from
`"{" +`
to
`Word("{[")`
and now everything works fine.
Do you think this could be a fix to the issue?
Thanks!

**Comment by kvs8** [comment]
Hello!
Yes, that's correct. Initially, only bodies consisting of an object were supported, and lists were overlooked. The solution you proposed addresses the issue, thank you!

This fix is now available in version `vedro-replay 1.2.0`.
Please note that internal dependencies have also been updated, which required minor adjustments in the generated scenario:
https://github.com/kvs8/vedro-replay/blob/main/vedro_replay/templates/scenario.py.j2#L3

Thank you very much for your contribution and engagement!
